docs/contributing/overview.md

```diff
@@ -12,16 +12,18 @@ search:
 
 # Contributing
 
-Thanks for considering contributing to Prefect!
+Thank you for considering contributing to Prefect.
+These instructions are for working with Prefect 2.
+For Prefect 3 see the [Prefect 3 documentation](https://docs.prefect.io).
 
 ## Setting up a development environment
 
 First, download the source code and install an editable version of the Python package:
 
 <div class="terminal">
 ```bash
-# Clone the repository
-git clone https://github.com/PrefectHQ/prefect.git
+# Clone the repository's 2.x branch
+git clone --branch 2.x https://github.com/PrefectHQ/prefect.git
```

Comment:
they wouldn't be committing to 2.x directly, so i think this is unnecessary, we should make a more normal recommendation
```suggestion
git clone https://github.com/PrefectHQ/prefect.git
```